lib/matplotlib/backends/backend_nbagg.py

```diff
@@ -98,6 +103,7 @@ def connection_info():
     'zoom_to_rect': 'fa fa-square-o icon-check-empty',
     'move': 'fa fa-arrows icon-move',
     'download': 'fa fa-floppy-o icon-save',
+    'export': 'fa fa-file-picture-o icon-picture',
```

Comment:
Yes, it inserts a static image into the notebook.  I'm looking at this from the point of view where the canvas is one of many widgets in a gui, and you want explicit control over export behavior.